jaxrs/WFLY-8136_EAP7-1228_JAX-RS_annotations_in_CDI_Stereotype.adoc

```diff
@@ -0,0 +1,86 @@
+= WFLY-8136 / EAP7-1228 Allow JAX-RS annotations in CDI @Stereotype
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This feature provides support for JAX-RS annotations to be defined as a part of 
+CDI `@Stereotype` definitions. This will allow JAX-RS annotations like `@Produces` 
+or `@Consumes` to be reused by several JAX-RS resources or providers.
```

Comment:
> like `@Produce` or `@Consume`

Can you please provide full list of supported annotations here?

Reply:
Support of javax.ws.rs.NameBinding should be also clarified here ...

Reply:
NameBinding is targeting only annotation_type. Stereotypes may be used only on classes and producer methods and fields.

Reply:
Yes, user can create new annotation with NameBinding and user can try to use this new annotation on Stereotype ...